### ISSUE TITLE
AsyncIOBackend logs 'unclosed client session' error on each trigger()

### ISSUE THREAD

**Issue by zdexter** [pusher/pusher-http-python#108]
`pusher==1.7.3` and `aiohttp==2.3.9`

`pusher.trigger` any message, and you'll see the following error if you have logging enabled (this is reported as an error level event to any attached tracing software etc, so it generates a lot of spam):

```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f3c9004dba8>
```

**Comment by zdexter** [comment]
the logger that generates this error is the `asyncio` logger.

**Comment by zdexter** [comment]
https://github.com/aio-libs/aiohttp/issues/1175

**Comment by hot-leaf-juice** [comment]
Hi @zdexter 👋 

Sorry for the delay getting back to you on this. Thanks for the report and the link to the relevant issue upstream. I think this should do the trick #109 . Will aim to get this merged and a new version released asap.

**Comment by hot-leaf-juice** [comment]
Should be fixed in 1.7.4 :)

**Comment by zdexter** [comment]
thanks!
